lib/client.js

```diff
@@ -288,6 +288,17 @@ module.exports = class Client extends EventEmitter {
     concatArray(this.createRowStream(sql, opts), callback)
   }
 
+  queryAsync (sql, opts) {
+    return new Promise((resolve, reject) => {
+      this.query(sql, opts, (error, result) => {
+        if (error) {
+          return reject(error);
+        }
+        resolve(result);
+      });
+    });
+  }
```

Comment:
Can you fix the code style issues here? Run `npm test` to check, and `npx standard --fix` to automatically fix them.

Reply:
Done